index.html

```diff
@@ -41,6 +41,191 @@ <h1 class="primary-heading"> I’m Mariane Glad to see you! </h1>
 
         </ul>
     </section>
+
+    <section class="Projects">
+        <section class="Card-Works">
+            <header class="Snapshot">
+                <img src="./img/Snapshoot Portfolio.png" alt="snapshot poortfolio" class="snapshot">
+
+            </header>
+            <div class="info-block">
+                <h1 class="project-title">Tonic</h1>
+                <ul class="frame">
+                    <span class="client">
+                        CANOPY</span>
+                    <img src="./img/Counter.png" alt="" class="counter">
+                    <span class="role">Back End Dev</span>
+                    <img src="./img/Counter.png" alt="" class="counter">
+                    <span class="year">2015</span>
+    
+                </ul>
+                <span class="details">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </span>
+                <ul class="tags">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+            
+                    <button class="see-project">See Project</button>
+        
+            </div>
+        </section>
+
+        <section class="Card-Works">
+            <header class="Snapshot">
+                <img src="./Img/Snapshoot Portfolio (1).png" alt="snapshot poortfolio" class="snapshot">
+
+            </header>
+            <div class="info-block">
```

Comment:
I suggest that you kindly ensure that the cards use the same margins and paddings as the figma design. This is part of the project requirements
<img width="1449" alt="Screenshot 2023-01-13 at 18 49 08" src="https://user-images.githubusercontent.com/48546992/212361532-acfdce30-6c8a-4581-b5f4-934ace1b64c1.png">

This is your design

<img width="678" alt="Screenshot 2023-01-13 at 18 44 13" src="https://user-images.githubusercontent.com/48546992/212361247-0df427f9-d1fd-43b9-9ff1-ce1c3018c689.png">

this is the figma template
<img width="378" alt="Screenshot 2023-01-13 at 18 43 26" src="https://user-images.githubusercontent.com/48546992/212361307-02769c57-5b8d-4351-9544-c7527aa693b2.png">